sdk/log/exporter.go

```diff
@@ -0,0 +1,53 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package log // import "go.opentelemetry.io/otel/sdk/log"
+
+import (
+	"context"
+)
+
+// Exporter handles the delivery of log records to external receivers.
+type Exporter interface {
+	// DO NOT CHANGE: any modification will not be backwards compatible and
```

Comment:
Do we think users will take this as an indication of stability?  We could wait to add these disclaimers until we are about to make the stable release.  Thoughts?

Reply:
FWIW, this comment doesn't actually get rendered in the package site documentation (e.g. https://pkg.go.dev/go.opentelemetry.io/otel/sdk/trace#SpanExporter).

Reply:
@MrAlias Any perference?

Reply:
I don't have a strong opinion here. I guess to avoid developer confusion as we develop this it would be best to remove this and track its addition with an issue on the GA project.

Reply:
I created https://github.com/open-telemetry/opentelemetry-go/issues/5070 and I will remove these comments.